core/src/block.rs

```diff
@@ -16,7 +16,7 @@
 
 use super::invoice::Invoice;
 use crate::client::{EngineInfo, TermInfo};
-use crate::consensus::CodeChainEngine;
```

Comment:
Please fix the typo in the commit message.
`CodeChainEngineCompared with ConsensusEngine` -> `CodeChainEngine compared to ConsensusEngine`

Reply:
I fixed it